db/schema.rb

```diff
@@ -308,14 +307,22 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
I'm confused where this comes from, I'll clean it up tomorrow when I add the tests.